Gopkg.lock

```diff
@@ -247,6 +247,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "55e7e3ee6c34b0ff4a85023d5bc5df5ce40c46b9cd423f8937303f6c61db32fa"
+  inputs-digest = "51b533603727b61d2cbde29d146b7ba6e6f10728bfb38b9f20a97c040bed697d"
```

Comment:
Why did this change?

Reply:
Don't know. May be this cause i use dep ensure and dep prune. I can fork again and copy changes.

Reply:
Nothing changed in the vendor folder so this should not have changed